### PR TITLE
Added axis title plugin to plugins page.

### PR DESCRIPTION
**Pull request by astanb**
Hi, I wrote a simple axis title plugin, can you check it out and see if it's ok? I'm also not sure about how to embed the example of it on this plugins page so you might need to advise me on that.

Cheers,

Alex

### CODE REVIEW

File: site/data/pages/plugins.yml [hunk 1]
```diff
@@ -179,6 +179,24 @@ sections:
...
+                        The axis title plugin allows you to add simple titles to your axises.
+            - type: table
+              data:
+                rows:
+                  -
+                    - '<strong>Author:</strong>'
+                    - Alex Stanbury
+                  -
+                    - '<strong>Link:</strong>'
+                    - '<a href="https://github.com/alexstanbury/chartist-plugin-axistitle" target="_blank">chartist-plugin-axistitle</a>'
```
Comment (gionkunz) [inline site/data/pages/plugins.yml:205]: Hi there! Great to see a plugin for this! :-) There's an issue with the link https://github.com/alexstanbury/chartist-plugin-axistitle as your repository is named chartist-plugins-axistitle. Maybe you can rename your repository to chartist-plugin-axistitle so it's named the same way as the other plugins? No need for it actually... but the link needs to be fixed either way. Cheers

**Comment by gionkunz** [comment]
Hey very cool, thanks for the PR :-) For the live example just add an other item to your plugin section in the yaml file:

``` yaml
            - type: live-example
              data:
                id: example-plugin-pointlabel
                classes: ct-golden-section
                intro: >
                         The point label plugin draws labels on top of your line chart points
```

Then just make sure you have a file with the name of the id (example-plugin-pointlabel.js  in the case above) in the examples directory https://github.com/gionkunz/chartist-js/tree/develop/site/examples .

Also you need to add your plugin as bower dependency https://github.com/gionkunz/chartist-js/blob/develop/bower.json#L17 and add it into the layout (as we're not using wiredep or something similar) https://github.com/gionkunz/chartist-js/blob/develop/site/layouts/default.hbs#L71-L75

Hope that helps...
Cheers

**Comment by astanb** [comment]
Hi,

Thanks for getting back to me so quickly. I'll make those changes tomorrow
when I get to the office. Silly mistake about my repository name...I am
making a lot of spelling and grammar mistakes lately :o

Cheers,

Alex

**Comment by alexstanbury** [comment]
Closing to trigger rebuild.

**Comment by gionkunz** [comment]
Hey there. Looks fantastic now! Travis just threw up. Did you publish to bower? Also make sure you have a tag 0.0.1 in your repository as bower uses git taga to obtain the package.

**Comment by astanb** [comment]
All sorted now, I've not done this stuff before as I'm sure you can tell :P

On 25 June 2015 at 10:56, Gion Kunz notifications@github.com wrote:

> Hey there. Looks fantastic now! Travis just threw up. Did you publish to
> bower? Also make sure you have a tag 0.0.1 in your repository as bower uses
> git taga to obtain the package.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gionkunz/chartist-js/pull/367#issuecomment-115195113.

## 

## Alex Stanbury

07973 832295
alex@alexstanbury.com

## uk.linkedin.com/in/astanbury

**Comment by gionkunz** [comment]
Ok Travis said its fine now :-) cool! Ill check it out tonight and publish the docs asap. Cheers and great plugin! I was waiting for this for somw time now, good stuff.

**Comment by astanb** [comment]
You're welcome, thanks for creating Chartist - it's great!

I am going to build a chart legend plugin next, I'll create a new PR once I think it's ready for you to have a look.

**Pull request merged**